src/Microsoft.Dnx.Host/BootstrapperContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Dnx.Host
+{
+    public class BootstrapperContext
+    {
+        public string OperatingSystem { get; set; }
+
+        public string OsVersion { get; set; }
+
+        public string Architecture { get; set; }
+
+        public uint NumProcessors { get; set; }
+
+        public string RuntimeDirectory { get; set; }
+
+        public string ApplicationBase { get; set; }
+
+        // TargetFramework?
```

Comment:
@anurse: I am thinking about passing TargetFramework in this struct instead of as a separate parameter - sounds good?